src/cortxfs/cortxfs_fops.c

```diff
@@ -186,25 +194,26 @@ static inline ssize_t __cfs_write(struct cfs_fs *cfs_fs, cfs_cred_t *cred,
 	RC_WRAP_LABEL(rc, out, dstore_pwrite, obj, offset, count,
 		      bs, (char *)buf);
 
-	RC_WRAP(cfs_getattr, cfs_fs, cred, &fd->ino, &stat);
-	RC_WRAP_LABEL(rc, out, cfs_amend_stat, &stat, STAT_MTIME_SET| STAT_CTIME_SET);
+	RC_WRAP_LABEL(rc, out, cfs_amend_stat, stat,
```

Comment:
Should the amend should happen after changing the stat struct ?

Reply:
Order doen't matter as `cfs_amend_stat` will update the `stat` pointer we are passing as an argument and then later on same pointer we are trying to update the size. Nothing will be written to disk using `cfs_amend_stat`.